### ISSUE TITLE
NTR: Bile alcohols high in urine (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8839]
New term request Bile alcohols high in urine (elthat_101120220915):   UMLS: 1859267

**Comment by pnrobinson** [comment]
@MickeySegal

is this Cerebrotendinous xanthomatosis?

**Comment by MickeySegal** [comment]
Yes.  Table 1 in https://www.ncbi.nlm.nih.gov/books/NBK1409/

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001007
- label: Elevated urinary bile alcohol level
- parents: [Abnormal urinary organic compound level (HP:6000531)](https://hpo.jax.org/app/browse/term/HP:6000531)
- pmid: PMID:20301583
- def: The amount of bile alcohol in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8839
- synonyms:
